.gitignore

```diff
@@ -1,5 +1,4 @@
-/crafty.js
-/crafty-min.js
+dist/
```

Comment:
Don't we need to include this for the files to actually be accessible to bower?

The whole root issue is that (contrary to standard practices) bower *requires* you to include the release artifacts in the repository  If we don't want to do that, the alternative is to have a separate repo for distribution.